packages/account/src/Components/address-details/address-details.tsx

```diff
@@ -141,7 +141,7 @@ const AddressDetails = ({
     };
 
     const handleCancel = (values: FormikValues) => {
-        const current_step = (getCurrentStep?.() || 1) - 1;
+        const current_step = (getCurrentStep?.() ?? 1) - 1;
```

Comment:
@jim-deriv Do we intend to use 0 value here if the returned value is 0? if not, it's better to keep || . :) same for line 167